server/src/action/auth/login.js

```diff
@@ -20,6 +20,8 @@ class Login extends AbstractAction {
 
     req.session.user = user
 
+    delete user.password
```

Comment:
As I said previously, this is a black-list approach : you remove the bad elements from user instead of keeping the good ones. The problem with this approach is it isn't very much future-proof : if new secret elements are added to `user`, they would leak here, which isn't a good thing.